config/main.py

```diff
@@ -2483,16 +2498,23 @@ def reload(ctx, no_dynamic_buffer, dry_run, json_data, ports):
             config_db = ConfigDBConnector(
                 use_unix_socket_path=True, namespace=ns
             )
+            static_config_db = ConfigDBConnector(
+                use_unix_socket_path=True, namespace=ns
+            )
 
         config_db.connect()
+        static_config_db.db_connect("STATIC_CONFIG_DB")
 
+        # TODO, HUA: remove buffer data from config DB after migration finish
         if not no_dynamic_buffer and asic_type in vendors_supporting_dynamic_buffer:
             buffer_template_file = os.path.join(hwsku_path, asic_id_suffix, "buffers_dynamic.json.j2")
             buffer_model_updated |= _update_buffer_calculation_model(config_db, "dynamic")
+            buffer_model_updated |= _update_buffer_calculation_model(static_config_db, "dynamic")
```

Comment:
_update_buffer_calculation_model is not necessary, because it's update device metadata -> buffer_model. will revert this change later.